contracts/swap/src/swap.rs

```diff
@@ -260,6 +261,10 @@ pub fn handle_atomic_order_reply(
     let swap_results_json = serde_json_wasm::to_string(&swap_results).unwrap();
     let swap_event = Event::new("atomic_swap_execution")
         .add_attribute("sender", swap.sender_address.to_owned())
+        .add_attribute("swap_input_amount", swap.input_funds.amount)
+        .add_attribute("swap_input_denom", swap.input_funds.denom)
+        .add_attribute("refund_amount", swap.refund.amount.to_owned())
+        .add_attribute("refund_denom", swap.refund.denom.to_owned())
```

Comment:
if `refund_denom` == `swap_input_denom`, we don't need to emit that `refund_denom` field

Reply:
removed